gptqmodel/models/base.py

```diff
@@ -80,6 +81,8 @@ def __init__(
         quantize_config: QuantizeConfig,
         qlinear_kernel: nn.Module = None,
         load_quantized_model: bool = False,
+        trust_remote_code: bool = False,
+        original_model_name_or_path: str = None,
```

Comment:
Why do we need to pass `original_model_name_or_path`? Isn't this always just model_path_or_name?

Reply:
update to model_name_or_path
cls or quantize_config not set model_path_or_name, save_quantize can not get original_model_path_or_name